config/hathitrust_config.rb

```diff
@@ -11,33 +11,26 @@ def volume_writer
           subsets.map do |subset|
             [subset, subset_volume_writer(subset)]
           end
-          .concat([superset, superset_volume_writer])
+          .concat([[superset, superset_volume_writer]])
           .to_h
       end
 
       def src_path_resolver
         @src_path_resolver ||=
-          src_parent_dir.map do |profile, dir|
-            [profile, PairtreePathResolver.new(Pathname.new(dir))]
-          end
-          .to_h
+          PairtreePathResolver.new(Pathname.new(src_parent_dir))
       end
 
       def volume_repo
         @volume_repo ||=
           subsets.map do |subset|
             [subset, subset_volume_repo]
           end
-          .concat([superset, superset_volume_repo])
+          .concat([[superset, superset_volume_repo]])
```

Comment:
Yeah, this is a bit confusing and I already mentioned it to @malakai97. Added an issue #55